firebase-firestore/ktx/src/androidTest/java/com/google/firebase/firestore/IntegrationTestUtil.kt

```diff
@@ -131,3 +149,10 @@ inline fun <reified T : Exception> testAssertThrows(
     val exception: T = Assert.assertThrows(T::class.java) { runnable() }
     return Truth.assertThat(exception)
 }
+
+/**
+ * makes the class can be converted to Gson JsonStr with empty constructor.
+ */
+abstract class ToGsonStringAble {
+    operator fun invoke(): String = Gson().toJson(this)
```

Comment:
Can I use this to help me assert two classes' instances can be converted to the same json string?